relnotes

```diff
@@ -327,7 +327,7 @@ generate_notes () {
   range="$start_tag..$release_tag"
 
   if [[ $release_tag =~ ${VER_REGEX[release]} ]]; then
-    changelog_file="CHANGELOG-${BASH_REMATCH[1]}.${BASH_REMATCH[2]}.md"
+    changelog_file="CHANGELOG/CHANGELOG-${BASH_REMATCH[1]}.${BASH_REMATCH[2]}.md"
```

Comment:
@hoegaarden PTAL, I think this is the only location which needs to be changed here for the current patch release process.